datacapturing/src/main/java/de/cyface/datacapturing/PongReceiver.java

```diff
@@ -19,15 +19,17 @@
 import android.util.Log;
 
 import androidx.annotation.NonNull;
+import de.cyface.datacapturing.backend.DataCapturingBackgroundService;
+import de.cyface.synchronization.BundlesExtrasCodes;
 
 /**
- * A <code>BroadcastReceiver</code> and sender that send a <code>MessageCodes.PING</code> event to the system and
- * expects to receive a <code>MessageCodes.PONG</code> event if the background service is running. If not a timeout will
- * tell the caller, that the service is not running.
+ * A {@link BroadcastReceiver} and sender that send a {@link MessageCodes#GLOBAL_BROADCAST_PING} event to the system and
```

Comment:
this link will not work in Javadoc since it references an external class. Please use `<code>` Tags here.